### DISCUSSION TITLE
Compatibility with Combat Utility Belt's hide actor name feature

### DISCUSSION THREAD

**Discussion by j01101111sh** [mclemente/healthEstimate#27] [Ideas]
If you have Combat Utility Belt (CUB) and you enable the feature to hide actor names in initiative and chat, the name is displayed in combat when the actor takes damage that causes it to change the health estimate (e.g. badly injured -> near death). It would be nice if this module could hide actor names also - either through a standalone setting or by piggybacking on CUB's existing setting.

Example showing name displayed in health estimate update but not in health estimate update:
![image](https://user-images.githubusercontent.com/87207916/125131549-8edf1e00-e0c8-11eb-801b-0b1c6440b422.png)

**Comment by mclemente** [comment]
Check the Health Estimate Macros compendium. The Hide Estimates macro will make the target unknown.

**Reply by j01101111sh** [reply]
That looks a little different than what I'm suggesting. I think the macro hides the estimate entirely; what I'm suggesting is the ability to hide just the actor's name. For example, you have a combat where you don't want the players to know the name of the enemy (for story reasons) but you still want them to know the enemy is looking injured.

**Reply by mclemente** [reply]
I see your point. I've added a macro for that too.

**Reply by LowellRice** [reply]
Just found this thread looking to solve the same problem. How do I find/use the macro you're talking about?
